frame/support/procedural/src/construct_runtime/mod.rs

```diff
@@ -178,7 +178,15 @@ pub fn construct_runtime(input: TokenStream) -> TokenStream {
 				.and_then(|_| construct_runtime_final_expansion(explicit_decl)),
 	};
 
-	res.unwrap_or_else(|e| e.to_compile_error()).into()
+	let res = res.unwrap_or_else(|e| e.to_compile_error());
+
+	let res = expander::Expander::new("construct_runtime")
+		.dry(std::env::var("SP_API_EXPAND").is_err())
```

Comment:
```suggestion
		.dry(std::env::var("FRAME_EXPAND").is_err())
```

Or something similar. The current naming was "exclusively" for `sp-api`.

Reply:
Yeah, I did not want to introduce the new one.